bot/utils.py

```diff
@@ -12,6 +31,14 @@ def new_ticket(description, title, user_id):
     return new
 
 
+def new_user(user_uid, first_name, last_name):
```

Comment:
Вот здесь можно заанотировать, что возвращаемый тип это твой    UserDict
```suggestion
def new_user(user_uid, first_name, last_name) -> UserDict:
```